Rakefile

```diff
@@ -35,14 +35,19 @@ task(:generate_team_table) do
 
   contributors.keys.shuffle.each do |github_user|
     user_content = contributors[github_user]
+    github_user_name = user_content['name']
+    github_user_id = github_user_name.downcase.gsub(' ', '-')
+    github_profile_url = "https://github.com/#{github_user}"
 
     content << "<tr>" if counter % number_of_rows == 0
-    content << "<td>"
-    content << "<img src='https://github.com/#{github_user}.png?size=200' width=140>"
+    content << "<td id='#{github_user_id}'>"
```

Comment:
you could use github_user instead of constructing the id from the name.

Reply:
It doesn't work, because the anchor is automatically created by GitHub with that id as `href="#id"`. You can temporarily see this [here](https://github.com/revolter/fastlane/blob/693af3a54cedc2eb6830e5b59abfd6474b30e42a/README.md#j%C3%A9r%C3%B4me-lacoste) _(until I delete that branch)_.

Reply:
I also tried [here](https://github.com/revolter/fastlane/blob/656a4f005c134faad003e5a0980bf1ef8a709574/README.md#j%C3%A9r%C3%B4me-lacoste) to create that anchor myself, but GitHub still creates its own anchor.